targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_14_2/libraries/util/nrf_assert.h

```diff
@@ -72,6 +72,11 @@ extern "C" {
  * @param line_num The line number where the assertion is called
  * @param file_name Pointer to the file name
  */
+
+#if !defined(NDEBUG) && !defined(DEBUG_NRF_USER)
+#define DEBUG_NRF_USER
+#endif
```

Comment:
Sorry, I should have mentioned this sooner but I didn't expect this PR to spill into the new SDK 14.2 but I guess the only modification is this activation flag?

Do you mind putting the define into the config files instead please? I've already made configuration changes there so you wouldn't have to make modifications to the 14.2 SDK.

https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52832/config/sdk_config.h
https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/config/sdk_config.h

Reply:
As far as I know there's no way to link the nrf assert enable to !NDEBUG like this in the config system?

Reply:
Right, I meant the `sdk_config.h` files I linked. They contain SDK `#define` values and those three lines should work from there.